common/lib/xmodule/xmodule/js/src/capa/display.coffee

```diff
@@ -49,6 +52,12 @@ class @Problem
     @$('.clarification').blur (ev) =>
       window.globalTooltipManager.hide()
 
+    @$('.review-btn').focus (ev) =>
```

Comment:
We will want to add automated test coverage for this-- I believe it is explicitly called out in subsequent stories, so no worries now.